.github/workflows/reusable_consistency_proof.yaml

```diff
@@ -0,0 +1,72 @@
+# Copyright 2022 The Sigstore Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# TODO: Add flags for identities to monitor, persist identities file, file issue when identities are found
+
+name: Consistency Proof Template
+
+on:
+  workflow_call:
+    inputs:
+
+permissions:
+  contents: read
+
+env:
+  UPLOADED_LOG_NAME: checkpoint
+  LOG_FILE: checkpoint_log.txt
+
+jobs:
+  check:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@93ea575cb5d8a053eaa0ac8fa3b40d7e05a33cc8 # v2.4.0
```

Comment:
@asraa says you need to hardcode the reusable workflow repository, else it checks out the caller workflow repository 

For repo ref: You need to use OIDC token to find the reference for the workflow that was specified in the caller. Ask Asra for the GHA for doing this

Reply:
https://github.com/slsa-framework/slsa-github-generator/tree/main/.github/actions/detect-workflow